config/routes.rb

```diff
@@ -167,15 +168,15 @@
         get '/explain-breaks' => 'work_history/breaks#edit', as: :work_history_breaks
         post '/explain-breaks' => 'work_history/breaks#update'
 
-        get '/explain-break/new' => 'work_history/break#new', as: :new_work_history_break
+        get '/explain-break/new' => 'work_history/break#new', as: :work_history_break
```

Comment:
Might be missing something here, but `new_work_history_break` seems like the appropriate path helper?

Reply:
Yea you're right